### PR TITLE
refactor: some refactorings to make sure the user gets its data

### PR DESCRIPTION
**Pull request by chmette**
Small changes from today mornings 
- insert next again 
- refactor seed file
- adjust schema

### CODE REVIEW

File: src/seed/seedUserDB.js [hunk 1]
```diff
@@ -76,8 +76,8 @@ console.log(`build script `);
         password: 'psw123',
       };
       console.log(`User ${userData.email} has been `);
-      const user = new User(userData);
-      return user.save();
+      const user = User.create(userData);
+      return user;
```
Comment (AdrianValdes) [inline src/seed/seedUserDB.js:80]: Could you send what you found on this? I have been reading and apparently `Model.create()` is the shorthand from 
```
 const user = new User(userData);
      return user.save();
```

File: server.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ app.get('/', (req, res) => {
 });
 
 // Errorhandler
-app.use((err, req, res) => {
+app.use((err, req, res, next) => {
   res.status(err.status || 500).send({
```
Comment (AdrianValdes) [inline server.js:55]: yes, I already put it when you solve it.

File: src/models/User.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ const UserSchema = new Schema(
     password: { type: String, required: true },
     avatar: {
       type: String,
-      required: false,
+      required: true,
       default: 'http://localhost:5000/statics/avatar/dog.png',
```
Comment (AdrianValdes) [inline src/models/User.js:15]: Do wee need this required? It is being created anyways because we have a default. I just remove the required field.

**Review by AdrianValdes — COMMENTED** [review]
Hey, look at the comments, I think we are good with what we did together regarding these issues in the morning

File: src/seed/seedUserDB.js (hunk 1 above)
Comment (chmette) [reply src/seed/seedUserDB.js:80]: I did not read about  it in detail, we use it in the controller and i thought its shorter and creates all the defaults as well, that is why i use it. it works :)

File: src/models/User.js (hunk 3 above)
Comment (chmette) [reply src/models/User.js:15]: then we dont need it in the base url as well?

Comment (chmette) [reply src/models/User.js:15]: so we could delete this branch ?

Comment (AdrianValdes) [reply src/models/User.js:15]: Yes, we can close it.

**Pull request closed without merging**